### DISCUSSION TITLE
Full-duplex (built-in) i2s audio on esp32

### DISCUSSION THREAD

**Discussion by Gaai** [pschatzmann/arduino-audio-tools#867] [Q&A] [answered]
Anyone experience with using both built-in adc and built-in dac at the same time on esp32?
Is it possible? Do they need to be installed on different ports? Probably right? Else streams mix up or something? 
Anything else I should know beforehand?

Appreciate the headsup!

Kind regards,

**Comment by pschatzmann** [comment, marked as answer]
Both work only on port 0, so you can't use them at the same time!

**Reply by Gaai** [reply]
Wah that's a pity. Thanks for the fast response!
